src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -177,7 +177,7 @@ Wherever you called `Amplify.configure({ aws-exports });` previously (usually in
 </Columns>
 
 <Callout>
-  `Amplify.configure()` will now accept either the config JSON file or a strongly typed configuration object. Therefore, if you need to add additional configuration, you will call configure twice: once with the contents of **amplifyconfiguration.json**, and then again using `Amplify.getConfig()` plus any additions. Keep in mind that any call to configuration will fully override previous configurations, so pay special attention to nested configurations.
+  `Amplify.configure()` will now accept either the config JSON file or a strongly typed configuration object. You can use the `parseAmplifyConfig` util to parse the contents of config JSON file **amplifyconfiguration.json** into the strongly typed configuration object and modify or override properties as needed. Keep in mind that any call to configuration will fully override previous configurations, so pay special attention to nested configurations.
```

Comment:
Realize that you are just modifying existing content, but we should avoid documentation that implies the reader has historical context in to previous behavior.

By using verbiage that includes "now accepts" we make an assumption that our reader has awareness in to the behavior before "now", which may or may not be true, but does not provide any value for them regardless, and reads as confusing.

Additionally, stating the return of `parseAmplifyConfig`  is strongly typed is a misnomer unless the return value accurately describes the config, which is the loosely typed `ResourcesConfig`. IMO noting the typing here is unnecessary and should be omitted

Reply:
This callout comes under the [troubleshooting/migrate-from-javascript-v5-to-v6](https://docs.amplify.aws/gen1/react/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/#step-3-upgrade-amplify-cli-version-and-configuration-file:~:text=Amplify.configure(),to%20nested%20configurations.) section. I believe the assumption here is the customer understands how V5 works and is looking to understand the diff.

But yes I agree we could rephrase this, maybe refer to ResourceConfig as just a typed object and omit "strongly", updated.